stack-academy/recipes/role-vagrant.rb

```diff
@@ -2,13 +2,33 @@
 include_recipe 'ies-mysql'
 include_recipe 'ies-mysql::dev'
 include_recipe 'ies::role-phpapp'
-include_recipe 'nginx-app::vagrant'
+
+node['vagrant']['applications'].each do |app_name, app_data|
```

Comment:
cc @gilleyj — I think this is the culprit.

I am running the VM locally to test and I am gonna run a deployment on playground to ensure I broke nothing.